cmd/cli/commands/login.go

```diff
@@ -56,6 +48,15 @@ var loginCmd = &cobra.Command{
 				os.Exit(1)
 			}
 
+			// mark key as default if we can decrypt it with given
+			if err := ks.SetDefault(addr); err != nil {
+				cmd.Printf("Given address is not present into keystore.\r\nAvailable addresses:\r\n")
```

Comment:
"into" means direction, not position.
I know that it was there previously, but as far as you move this code - better to fix it.